lib/nodejs/supervisor.ex

```diff
@@ -29,38 +29,35 @@ defmodule NodeJS.Supervisor do
     Supervisor.stop(__MODULE__)
   end
 
-  defp to_transaction(module, args, opts) do
+  defp run_in_transaction(module, args, opts) do
     timeout = Keyword.get(opts, :timeout, @timeout)
     binary = Keyword.get(opts, :binary, false)
 
     func = fn pid ->
       try do
-        GenServer.call(pid, {module, args, binary}, timeout)
+        GenServer.call(pid, {module, args, binary, timeout}, timeout)
```

Comment:
Just to make this a bit easier to pass around, I think we should just go ahead and pass `opts` through. So we'd do something like:

```
GenServer.call(pid, {module, args, opts}, timeout)
```

This would make it a bit easier in the future if we need to move this further down the pipe.

Reply:
Right, makes a lot of sense to me. I'll change this :+1:

Reply:
I changed it to a keyword list now